package.json

```diff
@@ -39,43 +39,49 @@
     "postpublish": "node ./tasks/update-gh-pages.js"
   },
   "dependencies": {
+    "@terrestris/eslint-config-typescript-react": "^2.0.0",
```

Comment:
can be moved to `devDependencies` I think

Reply:
Right, fixed with 1fe2c12